### ISSUE TITLE
Enabling or disabling app clears header menu

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/dashboard#10]
Every time you enable or disable the app this happens:
![screenshot from 2017-08-02 20-32-43](https://user-images.githubusercontent.com/925062/28888690-d5c4da18-77c1-11e7-9b4e-15ac3521ce7a.png)
You need to manually refresh to fix it.

cc @juliushaertl @nextcloud/javascript

**Comment by MorrisJobke** [comment]
> Every time you enable or disable the app this happens:

What happens? The app menu hides?

**Comment by juliushaertl** [comment]
@jancborchardt Cannot reproduce on 12.0.1 RC4 or is this happening on master?

**Comment by juliushaertl** [comment]
Looks like there is some issue with the order of -1 that the dashboard app uses. Changing that to 0 makes it work again.

**Comment by MorrisJobke** [comment]
@juliushaertl Could you open a PR?

**Comment by juliushaertl** [comment]
@MorrisJobke I just wanted to have another look why things break. Changing to 0 makes the app appear after files, which is not wanted I guess.

**Comment by MorrisJobke** [comment]
> @MorrisJobke I just wanted to have another look why things break. Changing to 0 makes the app appear after files, which is not wanted I guess.

But I checked and it is simply a normal "Which is higher" comparison in the "proceedNavigation". Where does it appear in the normal menu with the -1?

**Comment by jancborchardt** [comment]
Currently it does appear before Files. But as said only after a refresh. :)

**Comment by lumenrobbin** [comment]
![image](https://user-images.githubusercontent.com/9877294/34428488-c74a483c-ec7f-11e7-8dc7-2da859eaf275.png)
i refresh this button save, but cannot save. anyone help?

**Comment by jancborchardt** [comment]
@lumenrobbin that's a separate problem, please open a new issue. :)

**Comment by daita** [comment]
@juliushaertl any update on this issue ?

**Comment by juliushaertl** [comment]
@daita Nope but I'll have a look at it again.
